src/backend/src/routes/tasks.routes.ts

```diff
@@ -20,4 +21,6 @@ tasksRouter.post(
 );
 tasksRouter.post('/:taskId/edit-status', isTaskStatus(body('status')), TasksController.editTaskStatus);
 
+tasksRouter.post('/:taskId/delete', TasksService.deleteTask.arguments.taskId);
```

Comment:
You want to add a method in the controller that is responsible for deleting the task! This function will get the query args from the route as the taskId and pass it into the service function!